doc/governance/org-docs/CHARTER.md

```diff
@@ -0,0 +1,73 @@
+# Charter for the HoloViz Organization
+
+This is the organizational charter for the HoloViz Organization. The HoloViz Organization is an open-source and collaborative effort with a mission to develop solutions for a wide range of visualization problems in Python.
+
+## 1. Terminology
+- "HoloViz" or "HoloViz Community" or "Community": The entire HoloViz ecosystem of Users, Contributors, and Projects
+- "HoloViz Organization" or "Organization": The voting members of HoloViz Committees and Projects
+- "HoloViz GitHub Organization": The Github organization at https://github.com/holoviz
+- "HoloViz" repo: The specific repository at https://github.com/holoviz/holoviz
+- "HoloViz.org": The website at https://holoviz.org/
+
+## 2. Organization Steering Committee
+By adding their name to the [Steering Committee.md file](./STEERING-COMMITTEE.md), Steering Committee members agree to the policies in this document.
+
+**2.1 Purpose**. The Steering Committee will be responsible for all technical oversight, project approval and oversight, policy oversight, and trademark management for the Organization.
+
+**2.2 Composition**. The Steering Committee voting members are listed in the steering-committee.md file in the repository. Voting members may be added or removed by no less than 3/4 affirmative vote of the Steering Committee. The Steering Committee will appoint a Chair responsible for organizing Steering Committee activity. There is no upper limit on the number of Steering Committee members.
+
+## 3. Voting
+
+**3.1. Decision Making**. The Steering Committee will strive for all decisions to be made by consensus. While explicit agreement of the entire Steering Committee is preferred, it is not required for consensus. Rather, the Steering Committee will determine consensus based on their good faith consideration of a number of factors, including the dominant view of the Steering Committee and nature of support and objections. The Steering Committee will document evidence of consensus in accordance with these requirements. If consensus cannot be reached, the Steering Committee will make the decision by a vote.
```

Comment:
what kind of decisions need to be voted on?

Reply:
anything that doesn't meet consensus, or anything explicitly stated like new member admission